pdr_backend/predictoor/test/test_predictoor_agent.py

```diff
@@ -218,7 +218,6 @@ def mock_build(*args, **kwargs):  # pylint: disable=unused-argument
     with patch(f"{d}.PredictoorAgent.get_ohlcv_data", mock_get_ohlcv_data2), patch(
         f"{d}.AimodelFactory.build", mock_build
     ):
-
         # initialize agent
```

Comment:
you removed a couple of lines from this file, can you please undo that to make the PR lighter?

Reply:
I'm asking since there will probably be multiple rounds, multiple comments and checks at every step... so every file counts.

Reply:
I agree. I have cherry-picked everything into a new PR. 
https://github.com/oceanprotocol/pdr-backend/pull/1423

Resolving this conversation.